_posts/2021-07-01-pulpcore-3.14-release-announcement.md

```diff
@@ -0,0 +1,131 @@
+
+If you'd like an in-depth look at the motivations, benefits, and details of the new tasking system,
+read [Matthias Dellweg's blog post](https://pulpproject.org/2021/06/21/a-new-tale/) and also watch
+ a YouTube livestreamed interview with Matthias conducted by Pulp's Brian Bouterse:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YWKw4RYluPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+As part of the work to implement this feature, this release also introduces a new more "graceful" shutdown for Pulpcore
+workers [#8930](https://pulp.plan.io/issues/8930)
+
+Note that for the moment, this feature is in tech-preview.
+Please [let us know what you think](https://pulpproject.org/help/#github-discussions).
+
+### Sticking with the old tasking system?
+If you intend to keep using the old tasking system, you should configure the `USE_NEW_WORKER_TYPE`
+setting to `False` before upgrading. For more information about how to enable or disable the new
+tasking system, see the [documentation](https://docs.pulpproject.org/pulpcore/configuration/settings.html#use-new-worker-type).
+
+
+## New Orphan Cleanup Endpoint [#8658](https://pulp.plan.io/issues/8658)
+
+This release introduces the `/pulp/api/v3/orphans/cleanup/` endpoint.
+When you call this endpoint with `POST` and no parameters, it is the equivalent of calling.
+`DELETE /pulp/api/v3/orphans/`. Note that as part of this release,
+`DELETE /pulp/api/v3/orphans/` has been deprecated.
+
+You can specify a list of `content_hrefs` to delete using the optional parameter `content_hrefs`
+with list of content hrefs. In this case, only the specified content units will be deleted by the
+orphan cleanup.
+
+## Increased Retries on Failed Downloads [#8881](https://pulp.plan.io/issues/8881)
+
+The default behavior for retrying downloads from remote sources after an error occurs has been
+updated. Downloads will now be retried on a wider range of errors, for example, HTTP 500 and
+websocket errors. The default number of retries is 3.
+
+If you're a plugin writer, you also have the option to configure the number of retries for your
+plugin with the new `DEFAULT_MAX_RETRIES` field that has been added to the ``Remote`` base class.
+
+## Filter the task list with correlation ID [#8891](https://pulp.plan.io/issues/8891)
+
+When using the task list endpoint, you can now filter the list by correlation ID to see all tasks being emitted by a single user action.
+
+## Changes to Download Concurrency [#8897](https://pulp.plan.io/issues/8897)
+
+Previously, `download_concurrency` would be set to a default value upon creation. With this release,
+the default value is set to `NULL` and returns `NULL` when queried. However, a default value will
+still be used. This value comes from the Pulp global settings or can be set by the plugin writer.
+
+## Updates to the Plugin API
```

Comment:
I think all plugin API content could be split into its own blog post since it's for a different reader altogether. I'm merging this as is though because I'm excited to have it in the release announcement going out. Thank you!

Reply:
Thank you! I will do that in future then!

Reply:
I think it's good to have them in the same post, because the pluginwriter will be interested in both, core changes and plugin api related tasks. Also often a core change is related to a plugin api change, and announcing them in different documents makes it harder to interrelate.
Is there a way we can make it easier for the "normal" consumer to understand he can stop reading here?